resources/shellHelperFunctions.sh

```diff
@@ -90,16 +90,26 @@ if [[ -n "$TERM" ]] && [[ "$TERM" != "dumb" ]] && [[ "$TERM" != "unknown" ]]; th
     COLOR_BLUE=$(tput setaf 4)
     COLOR_YELLOW=$(tput setaf 3)
     COLOR_RESET=$(tput sgr0)
+    SETMARK='\x1b]1337;SetMark\x07'
```

Comment:
```suggestion
    # e.g. VSCode https://code.visualstudio.com/updates/v1_69#_setmark-sequence-support
    HEADING_SETMARK='\x1b]1337;SetMark\x07'
```

I didn't know what this did!

Thought we should make the variable name slightly less generic?